pkg/assembler/clients/operations/trees.graphql

```diff
@@ -298,48 +298,48 @@ fragment AllCertifyVEXStatement on CertifyVEXStatement {
   collector
 }
 
-fragment AllHasMetadata on HasMetadata {
-    id
-    subject {
-      __typename
-      ... on Package {
-        ...AllPkgTree
-      }
-      ... on Source {
-        ...AllSourceTree
-      }
-      ... on Artifact {
-        ...AllArtifactTree
-      }
-    }
+# fragment AllHasMetadata on HasMetadata {
```

Comment:
Why is this commented out?

Reply:
@arorasoham9 can you fix this comment and we should be ok to merge.

Reply:
fixed.

Reply:
ah so that is a bug. It is missing from the path queries here: https://github.com/guacsec/guac/blob/main/pkg/assembler/clients/operations/path.graphql

So it needs to be added like this:
```
    ... on HasMetadata {
      ...AllHasMetadata
    }
```

Added for the Path query, Neighbors query, Node query and Nodes query. That will resolve this error

Reply:
Fixed!